acquia_lift.admin.wizard.inc

```diff
@@ -1030,6 +1030,7 @@ function acquia_lift_personalize_campaign_wizard_submit(&$form, &$form_state) {
 
   // Run the base submit form handling.
   $agent_data = personalize_campaign_wizard_submit_base($form, $form_state);
+  $agent_data = personalize_agent_load($agent_data->machine_name);
   $agent_instance = personalize_agent_load_agent($agent_data->machine_name);
```

Comment:
Yeah... but we should probably still think about _personalize_agent_from_form_values given that there is never going to be that "options" form there anymore.  I think that part is just to be removed.  Want me to look more into it as a follow-up?